Tools/CMake/AMReX_Config.cmake

```
@@ -155,6 +155,54 @@ function (configure_amrex)
       endif()
    endif()
 
+   #
+   # Setup DPC++ -- for now simply add public compile flags
+   #
+   if (ENABLE_DPCPP)
+
+      target_compile_options( amrex
+         PUBLIC
+         $<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CXX_COMPILER_ID:Clang>>:-Wno-error=sycl-strict -fsycl -fsycl-unnamed-lambda>)
+
+      target_link_options(amrex PUBLIC -Wno-error=sycl-strict -fsycl -fsycl-unnamed-lambda -device-math-lib=fp32,fp64)
+
+      if (ENABLE_DPCPP_AOT)
+         if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+            ## TODO: use file(READ)
+            execute_process( COMMAND cat /sys/devices/cpu/caps/pmu_name OUTPUT_VARIABLE _cpu_long_name )
+         else ()
+            message(FATAL_ERROR "\nENABLE_DPCPP_AOT is not supported on ${CMAKE_SYSTEM_NAME}\n")
+         endif ()
+
+         string(STRIP "${_cpu_long_name}" _cpu_long_name)
+         if (_cpu_long_name STREQUAL "skylake")
+            set(_cpu_short_name "skl")
+         elseif (_cpu_long_name STREQUAL "kabylake")
+            set(_cpu_short_name "kbl")
+         elseif (_cpu_long_name STREQUAL "cascadelake")
+            set(_cpu_short_name "cfl")
+         else ()
+            message(FATAL_ERROR "\n AOT CPU ${_cpu_long_name} not yet supported\n")
```

Users love this trick ;-)
```suggestion
            message(FATAL_ERROR "\n Ahead-of-time compilation for CPU ${_cpu_long_name} is not yet supported\n"
            "Maybe set ENABLE_DPCPP_AOT to OFF?\n")
```